src/test/java/games/strategy/engine/chat/ChatTest.java

```diff
@@ -151,61 +155,57 @@ public void run() {
     // we need to wait for all the messages to write
     for (int i = 0; i < 10; i++) {
       try {
-        assertEquals(m_client1ChatListener.m_messages.size(), 3 * messageCount);
-        assertEquals(m_client2ChatListener.m_messages.size(), 3 * messageCount);
-        assertEquals(m_serverChatListener.m_messages.size(), 3 * messageCount);
+        assertEquals(client1ChatListener.messages.size(), 3 * messageCount);
+        assertEquals(client2ChatListener.messages.size(), 3 * messageCount);
+        assertEquals(serverChatListener.messages.size(), 3 * messageCount);
         break;
       } catch (final AssertionError afe) {
         ThreadUtil.sleep(25);
       }
     }
-    assertEquals(m_client1ChatListener.m_messages.size(), 3 * messageCount);
-    assertEquals(m_client2ChatListener.m_messages.size(), 3 * messageCount);
-    assertEquals(m_serverChatListener.m_messages.size(), 3 * messageCount);
+    assertEquals(client1ChatListener.messages.size(), 3 * messageCount);
+    assertEquals(client2ChatListener.messages.size(), 3 * messageCount);
+    assertEquals(serverChatListener.messages.size(), 3 * messageCount);
     client1.shutdown();
     client2.shutdown();
     // we need to wait for all the messages to write
     for (int i = 0; i < 10; i++) {
       try {
-        assertEquals(m_serverChatListener.m_players.size(), 1);
+        assertEquals(serverChatListener.players.size(), 1);
         break;
       } catch (final AssertionError e) {
         ThreadUtil.sleep(25);
       }
     }
-    assertEquals(m_serverChatListener.m_players.size(), 1);
+    assertEquals(serverChatListener.players.size(), 1);
     controller.deactivate();
     for (int i = 0; i < 10; i++) {
       try {
-        assertEquals(m_serverChatListener.m_players.size(), 0);
+        assertEquals(serverChatListener.players.size(), 0);
         break;
       } catch (final AssertionError afe) {
         ThreadUtil.sleep(25);
       }
     }
-    assertEquals(m_serverChatListener.m_players.size(), 0);
+    assertEquals(serverChatListener.players.size(), 0);
   }
 
   private static class TestChatListener implements IChatListener {
-    public List<INode> m_players;
-    public List<String> m_messages = new ArrayList<>();
-    public List<Boolean> m_thirdPerson = new ArrayList<>();
-    public List<String> m_from = new ArrayList<>();
```

Comment:
`m_thirdPerson` and `m_from` were not used in this test, so I just removed them.